File: frontend/src/app/log/models/log-model.ts
import { EventModel } from './event-model';
import { EmployeeModel } from './employee-model';
import { AdminModel} from './admin-model';
export interface LogModel {
    id: string;
    employee: EmployeeModel;
    admin: AdminModel;
    date_of_event: string;
    log_objects: EventModel[];

}

File: backend/src/main/java/com/rockhead/RockHead/log/LogModel.java
package com.rockhead.RockHead.log;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.rockhead.RockHead.admin.Admin;
import com.rockhead.RockHead.employee.Employee;
import lombok.Data;

import java.util.Date;
import java.util.List;

@Data
public class LogModel {
    private String id;

    private Employee employee;

    private Admin admin;

    @JsonProperty("date_of_event")
    private Date dateOfEvent;

    @JsonProperty("log_objects")
    private List<Event> logObjects;
}
